content/en/api/usage/usage_hosts.md

```diff
@@ -34,3 +34,9 @@ Get Hourly Usage For Hosts and Containers.
 * **`aws_host_count`**:
     Contains the total number of hosts that reported via the AWS integration (and were NOT running the Datadog Agent).
     When AWS or GCP hosts are also running the Datadog Agent, they are counted as Agent hosts, NOT as AWS or GCP.
+* **`alibaba_host_count`**:
```

Comment:
The `host_count` info needs to be updated as well [line 25 above https://github.com/DataDog/documentation/pull/6866/files#diff-2bd02b1dffb1190f6e72c9c1cca7466fR25] to mention that `alibaba_host_count` and `azure_host_count` are included as well.